scripts/windows/env.bat

```diff
@@ -0,0 +1,22 @@
+@echo off
+
+set WIN_SCRIPTS_DIR=%~dp0
+set SCRIPT_DRIVE=%~d0
+echo WIN_SCRIPTS_DIR=%WIN_SCRIPTS_DIR%
+echo SCRIPT_DRIVE=%SCRIPT_DRIVE%
+
+if /i "%LOCALBUILD%" equ "True" (
+    set PHP_BUILD_CACHE_BASE_DIR=C:\build-cache
+    set PHP_BUILD_OBJ_DIR=C:\obj
+    set PHP_BUILD_CACHE_SDK_DIR=C:\build-cache\sdk
+    set PHP_BUILD_SDK_BRANCH=php_src-2.2.1-dev
+    set PHP_BUILD_CRT=vs16
+    set PLATFORM=x64
+    set THREAD_SAFE=1
+    set INTRINSICS=AVX2
+    set PARALLEL= -j2
+    set OPCACHE=1
+
+    set CI=True
```

Comment:
In part of your review, you acknowledge these scripts may not run in github actions. In other places you do not.

In any case these scripts are replacing lots of manual commands, so I think this shows that we are running CI (unattended) scripts locally, rather than writing a specific purpose local script.

I Have every desire to see if I can help folks not use this for CI, but right now that and my own computer, are the only places it is used.